tests/e2e/pom.xml

```diff
@@ -0,0 +1,139 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019-2021 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-server</artifactId>
+        <groupId>org.eclipse.che</groupId>
+        <version>7.29.0-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <groupId>org.eclipse.che.e2e</groupId>
+    <artifactId>che-e2e</artifactId>
+    <name>Che e2e tests</name>
+    <build>
+        <finalName>e2e-war</finalName>
+        <plugins>
+            <plugin>
+                <groupId>com.mycila</groupId>
+                <artifactId>license-maven-plugin</artifactId>
+                <configuration>
+                    <useDefaultExcludes>false</useDefaultExcludes>
+                    <excludes>
+                        <exclude>**/.idea/**</exclude>
+                        <exclude>**/*.styl</exclude>
+                        <exclude>**/*.html</exclude>
+                        <exclude>**/*.ico</exclude>
+                        <exclude>**/*.ttf</exclude>
+                        <exclude>**/*.eot</exclude>
+                        <exclude>**/*.css</exclude>
+                        <exclude>**/*.woff</exclude>
+                        <exclude>src/app/colors/che-output-colors.constant.ts</exclude>
+                        <exclude>src/app/proxy/proxy-settings.constant.ts</exclude>
+                    </excludes>
+                </configuration>
+            </plugin>
+            <plugin>
+                <artifactId>maven-antrun-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>remove-old-resources</id>
+                        <phase>clean</phase>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <configuration>
+                            <target>
+                                <exec dir="${basedir}" executable="docker" failonerror="true">
```

Comment:
do we need docker to remove files ?

nodejs:10 is also EOL

Reply:
docker command replaced by bash, nodejs:10 replaced by nodejs:15